deploy/distrib/src/main/resources/tomcat/conf/server.xml

```diff
@@ -152,12 +152,11 @@
              Note: The pattern used is equivalent to using pattern="common" -->
 
         <!-- AccessLogValve below is commented out as it is not advisable for production environments -->
-        <!--
+        <!-- 
         <Valve className="org.apache.catalina.valves.AccessLogValve" directory="logs"
                prefix="localhost_access_log." suffix=".txt"
-               pattern="%h %l %u %t &quot;%r&quot; %s %b" />
+               pattern="%{yyyy}t-%{MM}t-%{dd}t %{HH}t:%{mm}t:%{ss}t.%{SSS}t %{Z}t %h %l %u &quot;%r&quot; %s %b" />

```

Comment:
the valve is commented by default. When the customer will activate it the message will be consistent with the new log messages:
2015-04-24 08:35:37.684 +0200 127.0.0.1 - - "GET /bonita/API/bdm/businessData/com.support.model.SupportCase?p=0&c=100&o=persistentId%3dDESC&q=getNonRejectedSupportCases HTTP/1.1" 200 4877